docs/extend.rst

```diff
@@ -59,6 +59,22 @@ Global extensions can also be disabled in the same way local extensions are
 
 .. _disabled: http://docs.chassis.io/en/latest/config/?highlight=disabled_extensions#extensions
 
+Updating extensions
+-------------------
+
+The Chassis extensions often received bug fixes, upgrades and enhancements and a developers Chassis project may require
+an extension to be updated if a provision is failing. Currently you need to update extensions manually. For example,
+let's pretend that the `Xdebug`_ extension has an update that you require. You would need to do the following:
+
+1. Open your terminal and navigate to your root Chassis folder.
+2. ``cd extensions\xdebug``
+3. ``git pull``
+4. ``vagrant provision``
+
+You can run steps 3 & 4 in each extensions directory and run step 4 to update all of your extensions.
```

Comment:
```suggestion
You can run steps 2 & 3 in each extensions directory and run step 4 to update all of your extensions.
```

Was that meant to be 2 & 3?

Reply:
Nah that was meant to be 3 & 4 because if I did 2 it'd be ``cd extensions\xdebug`` and that would be incorrect if they wanted to update say `tester`. Maybe I need to tweak the wording perhaps? To read makes my speak english good 😂

Reply:
Does step 4 requiring CD'ing back to the root?

Reply:
Negative. thankfully Vagrant is smart enough to keep walking up the directories until it finds the `.vagrant` folder.